cypress/e2e/2-Locators-Actions/layout.cy.js

```diff
@@ -0,0 +1,20 @@
+describe("Тест 1", () => {
+    it("Сторінка /pages/layout/stepper", () => {
+      cy.visit("/pages/layout/stepper");
+  
+      // Функція для перевірки тексту та натискання кнопки "NEXT"
+      function checkStepText(stepText) {
+        cy.get(".step-content").should("contain", stepText);
+        cy.get("body > ngx-app > ngx-pages > ngx-one-column-layout > nb-layout > div > div > div > div > div > nb-layout-column > ngx-components > ngx-stepper > nb-card > nb-card-body > nb-stepper > div.step-content > button:nth-child(3)").click();
```

Comment:
there is a better selector for it : 
```suggestion
        cy.get("h3:eq(0)").click();
```

Reply:
для цього тесту в мене селектор, який ти вказала - не працює
і відповідно всі інші зміни, додані до цього тесту - фейляться